projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/remote_data_job.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import os
+import time
+import uuid
+from enum import Enum
+from typing import Optional
+
+from taurus_datajob_api import ApiClient
+from taurus_datajob_api import Configuration
+from taurus_datajob_api import DataJobExecution
+from taurus_datajob_api import DataJobExecutionRequest
+from taurus_datajob_api import DataJobsExecutionApi
+from urllib3 import Retry
+from vdk.plugin.control_api_auth.authentication import Authentication
+
+log = logging.getLogger(__name__)
+
+
+class JobStatus(str, Enum):
+    """
+    Enum for the possible statuses a job execution can have.
```

Comment:
is that the very first enum we have? maybe have it in core, since it's going to be easier to keep consistent and up-to-date, for plugin usages?

Reply:
Probably not I added TODO to handle the duplication. I am going to go with rule of 3 on this and wait for now (https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)

Reply:
I agree. But This is just a prototype (pre-alpha) plugin. For a prototype, I don't think we should be changing things in core parts.